packages/server/graphql/public/mutations/upsertTeamPromptResponse.ts

```diff
@@ -20,10 +20,9 @@ const upsertTeamPromptResponse: MutationResolvers['upsertTeamPromptResponse'] =
 
   // VALIDATION
   if (inputTeamPromptResponseId) {
-    const responseId = TeamPromptResponseId.split(inputTeamPromptResponseId)
     const teamPromptResponse: TeamPromptResponse = await dataLoader
       .get('teamPromptResponses')
-      .load(responseId)
+      .load(inputTeamPromptResponseId)
```

Comment:
+1 I'm a little lost, where do we convert the ID now? wouldn't it be better to keep it as is to keep the PR as small as possible?

Reply:
The conversion (both joining and splitting) happens within the dataloader when the dataloader is used.  This reduces some complexity around the custom `reactables` loader, which would otherwise need to do some special casing to handle both number IDs and string IDs.  I originally tried doing things that way and leaving these IDs as-is, but IIRC it ended up breaking some existing abstractions and would've been difficult to refactor.